### PR TITLE
From iteration 7 - `cast` field with relation to `Celebrity` model

### PR DESCRIPTION
**Pull request by abernier**
Having that relation makes the lab really more interesting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -242,106 +242,136 @@ Now that we've done all this good work, it's time to do it all over again, but f
...
-## Iteration #7: The `Movie` Model
-Let's jump right in.
+## Iteration #7: The `movie` model
 
-First of all, we'll need to create the `Movie` model.
+Now when we've started all this good work, let's keep up strong and build all the routes for the *movie* model. But first, let's create the *movie* model.
 
-The `Movie` model should have:
+
+The `movie` model should have:
```
Comment (ross-u) [inline README.md:250]: Reference to the model name is in lowercase.

File: README.md [hunk 2]
```diff
@@ -242,106 +242,136 @@ Now that we've done all this good work, it's time to do it all over again, but f
...
 ### Steps we will follow in this iteration:
 
-Go back and review what you did to create the `Celebrity` model.  You'll need to create a file for the model, and in that file, you'll need to create a schema for the model as well.
-
-Once you've done that, go to your `seeds.js` file in the `bin/` folder and either delete or comment out the seeds you made before for your celebrities.
-Replace these seeds with seeds for fake movies.  If you don't delete/comment what you had before, when you run the seeds file with the `node` command in the terminal, it will create duplicates of all your celebrities.
-
-Afterward, check the database with the `mongo` command to confirm that your data was saved.
-
+Go back and review what you did to create the `celebrity` model.  You'll need to create a file for the model, and in that file, you'll need to create a schema for the model as well. Don't forget, you have to `module.exports` the `Movie` model.
```
Comment (ross-u) [inline README.md:258]: Reference to the model name is in lowercase.

File: README.md [hunk 3]
```diff
@@ -242,106 +242,136 @@ Now that we've done all this good work, it's time to do it all over again, but f
...
 ### Steps we will follow in this iteration:
 
-Go back and review what you did for the `Celebrity` model.  You'll need to:
-  - Create a route
-  - Use a database query to retrive the specific movie that was clicked by the user.
-  - Pass that movie into the view as a variable
-  - Create the view file
-  - In the view, display all the details of the movie.
-  - On the Movies index page, make each movie a link to its own details page.
+Go back and review how you did this for the `celebrity` model.  You'll need to:
```
Comment (ross-u) [inline README.md:292]: References to the models starts with a lowercase letter

**Review by ross-u — CHANGES_REQUESTED** [review]
Overall looking good. One thing that I noticed should be changed before the merge is that some of the references to the models start with a lowercase letter, but should instead start with an uppercase letter

**Comment by abernier** [comment]
> Overall looking good. One thing that I noticed should be changed before the merge is that some of the references to the models start with a lowercase letter, but should instead start with an uppercase letter

Hey Uros, I've validated all your 3 xhanges-requests ;) thx

**Comment by abernier** [comment]
@ross-u sorry, I guessed marking your request-changes as resolved would have done the commit: here it is now https://github.com/ironhack-labs/lab-mongoose-movies/pull/1607/commits/312e2288b0dce19c57190a2f28fd2185f764ec42 :)

We should be good

**Pull request merged**